### DISCUSSION TITLE
Mapping Megathread

### DISCUSSION THREAD

**Discussion by CubicRP** [CubicRP/Cubic#9] [Ideas]
Suggestions/issues regarding mapping.

(New interiors/maps, mapping issues etc - new scripts can be discussed in new discussions)

**Comment by UGotCuteToes** [comment]
add some vegetation to sandy shores instead of it just being a desert.

**Reply by dwreyyes** [reply]
We have been playing around with vegetation for most of North County.

**Comment by shifttt7** [comment]
Add a gym. For example, a city called scora network has a gym that makes u stronger have more stamina run quicker ETC. Obviously make it so its not to OP and people cant get too much stamina, strength and speed but ould be amazing for RP reasons.

**Comment by shifttt7** [comment]
These are some great things to add to make the RP much better. Here are all of them as clips i have made. https://1drv.ms/u/s!As8Jk3ulhDPThweGeC0wgKypZZ_K?e=Qj2tCo

**Reply by ChivSuroo** [reply]
The bowling and snooker is just a meh, mechanics are just terrible, with mechanics, we have a better system where everything is interactable, having people preview the parts themselves is just weird, as they wont have the experience to place them. 

Every part of the wheel, we need to place ourselves not just a stupid menu that previews for you.

**Comment by LucienLivinLarge** [comment]
Could we make fishing more accessible by adding a small public dock at the Boathouse in Sandy? This would also make Sandy more useful to live in.

Also is it possible for certain houses to add a private boat dock spawn like a garage if it is connected to the house?

**Comment by Hopeshii** [comment]
Repost:
Hello! I was wondering if we can maybe add a Piano mod. I think individuals who want to Role play as musicians will be able to show off their piano skills. In addition, they will have access to more musical instruments than just a DJ stand and a guitar. Plus, they have the chance to perform on the street.

Link:
https://store.rcore.cz/package/4707008

**Comment by Gibuso** [comment]
Idk why but i thought it would be interesting to have a metal fence between the prison and the freeway. Maybe it can follow a bit farther along the freeway. It would make it so people dont cut across the grass and make the area around prison feel a bit more secluded.
